elasticsearch/src/test/java/org/hibernate/search/elasticsearch/test/ElasticsearchAnalyzerDefinitionCreationIT.java

```diff
@@ -164,25 +164,25 @@ public void success_simple() throws Exception {
 							+ "},"
 							+ "'analyzerWithComplexComponents_WordDelimiterFilterFactory': {"
 									+ "'type': 'word_delimiter',"
-									+ "'generate_word_parts': '1',"
-									+ "'generate_number_parts': '1',"
-									+ "'catenate_words': '0',"
-									+ "'catenate_numbers': '0',"
-									+ "'catenate_all': '0',"
-									+ "'split_on_case_change': '0',"
-									+ "'split_on_numerics': '0',"
-									+ "'preserve_original': '1'"
+									+ "'generate_word_parts': 'true',"
```

Comment:
Yes it should, but it's out of my control. Elasticsearch just sends back a string, even though I originally sent a boolean.